lib/roo_on_rails/checks/sidekiq/sidekiq.rb

```diff
@@ -0,0 +1,48 @@
+require 'roo_on_rails/checks/base'
+require 'thor'
+
+module RooOnRails
+  module Checks
+    module Sidekiq
+      class Sidekiq < Base
+        WORKER_PROCFILE_LINE = "worker: RAILS_MAX_THREADS=$SIDEKIQ_DB_POOL_SIZE \
+                                DB_REAPING_FREQUENCY=$SIDEKIQ_DB_REAPING_FREQUENCY \
```

Comment:
What impact does `DB_REAPING_FREQUENCY` have? It's not documented in Sidekiq or in Rails (that I can find).